### ISSUE TITLE
Custom MET Significance Muon Calibration

### ISSUE THREAD

**Issue by mdhank** [UCATLAS/xAODAnaHelpers#1633]
The MET significance handler (https://github.com/UCATLAS/xAODAnaHelpers/blob/main/Root/METConstructor.cxx#L143) doesn't allow the muon calibration tool to be set, forcing it to use the default. This gives a warning at https://gitlab.cern.ch/atlas/athena/-/blob/master/Reconstruction/MET/METUtilities/Root/METSignificance.cxx#L157 , and may not always be the desired behavior.

I can look into how to update this. In principle we could just provide the same 3 options (m_calibrationMode , m_do2StationsHighPt, and m_isRun3Geo) that we provide to the Muon Calibrator to the MET Constructor.

**Comment by jbossios** [comment]
Can't we provide an instance of the muon calibration tool that we already initialized (in MuonCalibrator) such that we don't have to initialize the same tool again and also so we don't have to make sure the properties are set coherently?

**Comment by mdhank** [comment]
I agree that would be ideal. I'll have to look at how to implement it.

**Comment by mdhank** [comment]
I finally got around to looking into this. It looks like it may actually be fine as is. As far as I can tell, despite the warning in METSignificance.cxx, the calibration mode and other options are only used in the applyCorrection function of https://gitlab.cern.ch/atlas/athena/-/blob/master/PhysicsAnalysis/MuonID/MuonIDAnalysis/MuonMomentumCorrections/Root/MuonCalibTool.cxx . However, the MET significance only uses the expectedResolution function. I've run a few tests on some signal samples as well as 100k ttbar events using various calibration modes, and do not observe any difference in the MET significance.
